config/server-config.yaml.example

```diff
@@ -89,3 +89,13 @@ authz:
   auth:
     # Set to token for production
     method: none
+
+# Configuration for the healthcheck functionality
+# Defaults to disable if not defined
+#marketplace:
```

Comment:
If this only deploys one profile, should we rename this from `marketplace` to something like `default_policy` or `default_profile`?

Reply:
👍  to a different name -- I can imagine a bunch of _different_ features being marketplaces (for example, providers or possibly remediation methods), so I'd be more specific here and call these something like `managed_policy` or `managed_profiles`

Reply:
OTOH, we can always do a rename later, we just need to duplicate the keys in the config while we're doing the switchover.